scipy/integrate/_quadpack_py.py

```diff
@@ -700,6 +700,17 @@ def dblquad(func, a, b, gfun, hfun, args=(), epsabs=1.49e-8, epsrel=1.49e-8):
     >>> integrate.dblquad(f, 0, 1, lambda x: x, lambda x: 2-x, args=(3,))
         (0.9999999999999999, 1.6653345369377348e-14)
 
+    Compute the two-dimensional Gaussian Integral, which is the integral of the
+    Gaussian function :math:`f(x,y) = e^{-(x^{2} + y^{2})}`, over
+    :math:`(-\\infty,\\infty)`. That is, compute the integral
+    :math:`\\iint^{\\infty}_{-\\infty} e^{-(x^{2} + y^{2})} \\,dy\\,dx`:
+
+    >>> import numpy as np
+    >>> from scipy import integrate
```

Comment:
Please move the import to NumPy to the top and remove the import to integrate as already present.